Source/ThirdParty/libwebrtc/Source/third_party/libvpx/source/libvpx/vp9/encoder/vp9_encoder.c

```diff
@@ -6219,9 +6219,15 @@ static void update_level_info(VP9_COMP *cpi, size_t *size, int arf_src_index) {
   }
 
   // update compression_ratio
+#ifdef WEBRTC_WEBKIT_BUILD
```

Comment:
Not sure why this is in an `#ifdef`. Not harmful on any platform as far as I can tell.

Reply:
This is the style/pattern that's been used in `Source/ThirdParty/libwebrtc` for a while now.

Cc @youennf

Reply:
Probably because we don't want to add `<wtf/Platform.h>` dependencies to third-party sources.

Reply:
That argument is backwards! I am suggesting an unconditional change rather than one with an `#if`